Doxyfile

```diff
@@ -1,2688 +0,0 @@
-# Doxyfile 1.9.4
```

Comment:
The commit messages says "fix doxygen config and remove doxyfile" but it's only removing the Doxyfile. I was wondering if something is missing in this commit.

Reply:
Right. I mistakenly added that change to a wrong commit. Fixed now.